helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -531,9 +531,9 @@ image:
 
   otelcol:
     # The registry and name of the opentelemetry collector image to pull
-    repository: quay.io/signalfx/splunk-otel-collector
+    repository: quay.io/signalfx/splunk-otel-collector-dev
```

Comment:
Why are pointing to dev? wouldn't that be unstable?

Reply:
I'm not going to release the chart yet. I'll bump the version to the next stable one with the chart release. There is functionality here that depends on the latest changes in https://github.com/signalfx/splunk-otel-collector